darwin/torch/transforms.py

```diff
@@ -209,13 +215,6 @@ def __call__(self, image: PILImage.Image, target: TargetType) -> Tuple[PILImage.
             if num_keypoints:
                 keypoints = keypoints.view(num_keypoints, -1, 3)
 
-        keep = (boxes[:, 3] > boxes[:, 1]) & (boxes[:, 2] > boxes[:, 0])
-        boxes = boxes[keep]
-        classes = classes[keep]
-        masks = masks[keep]
-        if keypoints is not None:
-            keypoints = keypoints[keep]
```

Comment:
I believe this logic was added for cases with bboxes of either width or height of size 0px. Should we add it back? The easiest way to add it back is to compute `keep` using a `tmp_boxes` converted to `x,y,x,y` using `tmp_boxes[:, 2:] += boxes[:, :2]` and then filter `boxes` using `keep`.